### ISSUE TITLE
Support for grun

### ISSUE THREAD

**Issue by phdoerfler** [ihji/sbt-antlr4#30]
I'd like to test the generated parser using grun aka `org.antlr.v4.gui.TestRig`. Amongst other things it provides a simple GUI:

![Grun GUI](https://www.antlr.org/images/sample3.png)

This is probably trivial to do if the example on the [ANTLR website](https://www.antlr.org/index.html) is anything to go by:

```
$ antlr4 Expr.g4
$ javac Expr*.java
$ grun Expr prog -gui
100+2*34
^D
```

How about adding a task "grun" or perhaps `run in Antlr4` or `test in Antlr4` that starts grun? Thoughts?

**Comment by ihji** [comment]
Yes, definitely good to have. I will add this feature sooner or later but can't promise the exact date (PRs are always welcome btw 😄 )
